mvcc/kv.go

```diff
@@ -107,10 +107,12 @@ type KV interface {
 	// Write creates a write transaction.
 	Write() TxnWrite
 
-	// Hash retrieves the hash of KV state and revision.
-	// This method is designed for consistency checking purposes.
+	// Hash computes the hash of the KV's backend.
 	Hash() (hash uint32, revision int64, err error)
 
+	// HashByRev computes the hash of all MVCC keys up to a given revision.
```

Comment:
the hash of only mvcc keys up to a given revision (excluding KV backend metadata, leases, etc.).